results.tex

```diff
@@ -113,48 +113,43 @@ \subsection{French \gls{SFR} Deployment}
 supply.
  
 Used \gls{MOX} from an ASTRID reactor is 23.95\% plutonium
-in this simulation (see \cref{tab:comp}), whereas a fresh \gls{MOX} is 22\% plutonium.
+in this simulation (see \cref{tab:comp}), whereas fresh \gls{MOX} is 22\% plutonium.
 The plutonium breeding ratio in this simulation is thus assumed to be
 $\approx 1.08$.
 
-\Cref{fig:fuel} shows \gls{MOX} loaded in the \glspl{SFR} per month.
-The spikes are due to initial fuel demand for new deployment of \glspl{SFR}.
-The initial loading of new \glspl{SFR} are done with the \gls{MOX} created
-from legacy \gls{UNF}. Once the deployed \glspl{SFR} create enough amounts
- of extra plutonium, the legacy \gls{UNF} is no longer used. 
+\Cref{fig:fuel} shows \gls{MOX} loaded in the \glspl{SFR} per month.  The plot 
+has peaks during a period of aggressive deployment of \glspl{SFR} followed by 
+an equilibrium at 100 \gls{MTHM}. The peaks reoccur with the deployment of the 
+second generation of \glspl{SFR}.  The spikes are due to initial fuel demand 
+correspoding to these new deployments.  The initial cores loaded into new 
+\glspl{SFR} rely on the \gls{MOX} created from legacy \gls{UNF}. Once the 
+deployed \glspl{SFR} create enough extra plutonium, the legacy \gls{UNF} is no 
+longer used. 
```

Comment:
Is it worth mentioning that we do that by using the Cyclus' preference method/algorithm?

Reply:
Sure. I've added something along those lines. "Notably, this switch from a less preferred fuel origin to a more
preferred fuel origin is handled automatically within \Cyclus via user-defined preferences
within its dynamic resource exchange algorithm \cite{gidden_methodology_2016}."

Reply:
Yup! that looks good! thanks